Mod/sources/bookmarks-bar.css

```diff
@@ -19,12 +19,15 @@
 .bookmark-bar button {
   border-radius: var(--radius);
   padding: 0px 5px;
-  cursor: pointer;
 }
 /* used for padding next to favicon */
 .bookmark-bar button:not(.chevron):before {
   content: unset;
 }
+/* Align titles to look more centered (like Opera and other browsers) */
+.bookmark-bar button .title {
+  margin-bottom: -1px;
+}
```

Comment:
Does this have any effect on your system? I don't see any difference when I enable/disable this line.

Reply:
I just noticed an issue with scaling. It should be resolved by using -0.5px instead, but I can't test it at the moment.

Reply:
Turns out the issue is not with the titles, but with the images. I will have to find another solution. We can remove this for now.
```suggestion
```